pkg/storage/replica_command.go

```diff
@@ -3837,36 +3836,61 @@ func RelocateRange(
 func (r *Replica) adminScatter(
 	ctx context.Context, args roachpb.AdminScatterRequest,
 ) (roachpb.AdminScatterResponse, error) {
-	db := r.store.DB()
+	wantScatter := true
+	r.setWantScatter(wantScatter)
```

Comment:
This needs to get unset somewhere - e.g. after replicate queue processing or after giving up the lease